test/commons/text/unicode.js

```diff
@@ -69,6 +69,13 @@ describe('text.hasUnicode', function() {
 			});
 			assert.isTrue(actual);
 		});
+
+		it('returns true for a srting with characters in the Supplementary Private Use Area-A', function() {
+			var actual = axe.commons.text.hasUnicode('\u{F0019}', {
```

Comment:
Unfortunately, the unicode escape sequence is causing a syntax error in eslint.

Not seeing that in other projects using eslint, and the test and build steps happen just fine. Not sure what's going on there, and this will take some more debugging.

Reply:
Oh, just realized that the test directory is parsed as ES5. That explains it. 🤦‍♂